packages/modules/viewer-react/src/components/app-ui/EcsqlWidget.tsx

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { UiFramework } from "@itwin/appui-react";
+import { QueryRowFormat } from "@itwin/core-common";
+import {
+  Button,
+  ExpandableBlock,
+  LabeledTextarea,
+  Table,
+} from "@itwin/itwinui-react";
+import React, { useState } from "react";
+
+export default function EcsqlWidget() {
+  const [input, setInput] = useState<string>("");
```

Comment:
you shouldnt need to type the state explicitly for strings and booleans, those should be able to be inferred by the literals you initialize the state with